perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -157,6 +161,18 @@ def RobustRemoteCommand(self, command):
                     '--delete']
     return self.RemoteCommand(' '.join(wait_command), should_log=False)
 
+  def SetupRemoteFirewall(self):
+    """Sets up IP table configurations on the VM."""
+    if hasattr(self, 'allowed_ip_range') and self.allowed_ip_range:
```

Comment:
This doesn't really supply any additional security unless the cloud provider doesn't have a firewall - and many images dont have iptables on by default anyway. And I'm not sure how likely it is that someone will go to the trouble of modifying the other cloud providers to set allowed_ip_range - there's no documentation that this is what is needed, so unless they happened to see this PR, they will have to figure that out for themselves.